eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -161,12 +161,30 @@ function UpdateDocsMsMetadataForPackage($packageInfoJsonLocation, $packageInfo)
     $packageMetadata = $packageMetadataArray[0]
   }
 
+  # Copy package info file to the docs repo
+  $metadataMoniker = 'latest'
+  if ($originalVersion.IsPrerelease) {
+    $metadataMoniker = 'preview'
+  }
+  $packageMetadataName = Split-Path $packageInfoJsonLocation -Leaf
+  $packageInfoLocation = Join-Path $DocRepoLocation "metadata/$metadataMoniker"
+  $packageInfoJson = ConvertTo-Json $packageInfo
+  New-Item -ItemType Directory -Path $packageInfoLocation -Force
+  Set-Content `
+    -Path $packageInfoLocation/$packageMetadataName `
+    -Value $packageInfoJson
+
+  # Update Readme Content
+  if (!(Test-Path $packageInfo.ReadMePath)) {
```

Comment:
If $packageInfo.ReadMePath is ever null you also need to add a check for that because Test-Path will error if the value is null.